### PR TITLE
Diffuse base_weight description changed to match MaterialX EON implementation

### PR DESCRIPTION
**Pull request by portsmouth**
As discussed in https://github.com/AcademySoftwareFoundation/MaterialX/pull/1822#issuecomment-2127738467, the MaterialX implementation assumed that the `base_weight` acts as an overall multiplier of the EON BRDF. While in the OpenPBR spec, we had `base_weight` instead multiplying `base_color`.

In fact the MaterialX intepretation seems better, since then the `base_weight` functions as a linear modulation of the overall albedo, rather than doing something non-linear and dependent on the color. This PR makes the necessary changes to the wording/math of the spec to implement this.

image

image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1439,16 +1468,14 @@
 André Mazzone,
 Nikie Monteleone,
 Michael Nickelsky,
-Anton Palmqvist,
```
Comment (jstone-lucasfilm) [inline index.html:1442]: It looks like Anton and Nicolas were accidentally removed in this change!

Comment (portsmouth) [reply index.html:1442]: Will fix

Comment (portsmouth) [reply index.html]: Done in 8821f630914bb6eb0f952970479c1f6d3067b690 (also added some others).

**Review by jstone-lucasfilm — APPROVED** [review]
This looks great, thanks @portsmouth!

**Pull request merged**